core/class.go

```diff
@@ -56,7 +56,25 @@ type Cairo1Class struct {
 	Program         []*felt.Felt
 	ProgramHash     *felt.Felt
 	SemanticVersion string
-	Compiled        json.RawMessage
+	Compiled        CompiledClass
+}
+
+type CompiledClass struct {
+	Bytecode        []*felt.Felt    `json:"bytecode"`
+	PythonicHints   json.RawMessage `json:"pythonic_hints"`
+	CompilerVersion string          `json:"compiler_version"`
+	Hints           json.RawMessage `json:"hints"`
+	EntryPoints     struct {
```

Comment:
We can replace the `EntryPoints` field with the `External`, `L1Handler`, and `Constructor` fields.

Reply:
Do you mean, the `External`, `L1Handler`, and `Constructor` fields will be top-level fields of the `CompiledClass` struct?

Reply:
Yes.

Reply:
We can do so, but it will not match the definition reference and the response format.

Reply:
This is a core type, it does not need to match.